src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java

```diff
@@ -58,10 +65,20 @@ public boolean configure (StaplerRequest req, JSONObject formData) throws FormEx
         }
 
         public FormValidation doCheckGlobalName(@QueryParameter String globalName) {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
             if (globalName.matches("[a-zA-Z\\s]+")) {
                 return FormValidation.ok();
             }
             return FormValidation.warning("Name can only contain lowercase and uppercase letters and spaces");
         }
+
+        public ListBoxModel doFillCategoriesItems() {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```

Comment:
who can (or is supposed to) be able to use this?  
IIRC the `ViewsTabBar` is added to `ListViews` and those can be created and configured by regular users (so long as they have Item.Create/Item.Configure where they are to be used?

Reply:
I've added these permissions